src/applications/vaos/services/mocks/index.js

```diff
@@ -324,6 +329,7 @@ const responses = {
         { name: 'vaGlobalDowntimeNotification', value: false },
         { name: 'vaOnlineSchedulingVAOSServiceRequests', value: true },
         { name: 'vaOnlineSchedulingVAOSServiceVAAppointments', value: true },
+        { name: 'vaOnlineSchedulingFacilitiesServiceV2', value: true },
```

Comment:
Is there a reason this feature flag is named this way? I don't remember if we officially determined if we want to use `VAOS service` or `v2` as our naming convention.

Reply:
I named it this way because the calls that are behind this toggle aren't to the vaos service, they're to version 2 of the mobile facilities service.

I'd definitely name these flags more in line with the vets-api v2 convention we've been using in the code if we did them over again, but it seemed inaccurate to name this flag as "VAOS service" for calls that aren't from that service.

Reply:
Ah got it, 👍 .